BraveShareTo/tr.lproj/InfoPlist.strings

```diff
@@ -0,0 +1,2 @@
+/* Bundle display name */
+"CFBundleDisplayName" = "Open in Brave";
```

Comment:
@soner-yuksel can you give me Turkish translation for it? 
There's nothing in transifex